arch/arm/soc/nordic_nrf5/nrf52/Kconfig.defconfig.nrf52840_QIAA

```diff
@@ -14,7 +14,7 @@ config SOC
 
 config NUM_IRQS
 	int
-	default 46
+	default 48
```

Comment:
you commit message does not say why you increase this number (and how it relates to spi actually)

Reply:
updated commit message to indicate  number was increased as per documentation

The lower number of interrupts was causing a build error as the last SPI device has number 47